lib/app/features/chat/model/entities/reaction_data.c.dart

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:collection/collection.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:ion/app/exceptions/exceptions.dart';
+import 'package:ion/app/features/chat/model/entities/private_direct_message_data.c.dart';
+import 'package:nostr_dart/nostr_dart.dart';
+
+part 'reaction_data.c.freezed.dart';
+
+@immutable
```

Comment:
`@immutable` is redundant here

Reply:
Needed for == and hashtag override
![image](https://github.com/user-attachments/assets/90394041-4506-4211-82d5-fbb1975e1089)